src/autoRespawn/main.ts

```diff
@@ -381,30 +382,61 @@ async function handleMarkChangeEvent(event: MarkChangeEvent) {
       }
 
       const { position } = addedMark
+      const { coalition = addedMark.coalition } = command
+
+      let radius = SPAWNER_MINIMUM_DISPLACEMENT_METERS
```

Comment:
it's inaccurate to use the minimum displacement as the default radius here, those two things are not the same.

I expect that when I change the minimum distance between spawners that it DOES NOT effect the radius of the destroy command
If anything, those two things should be independently configurable.

Reply:
That's a good point, I just took what you used for the accuracy before and didn't think about it more. Will change it.

Reply:
It looks like I just did a copy paste here :o whoops. must have been lazy